<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newRegister</class>
 <widget class="QDialog" name="newRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>专业：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择专业</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>计算机科学与技术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>软件工程</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物联网工程</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>网络工程</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>其他</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
